code/modules/projectiles/guns/projectile/automatic/ak47.dm

```diff
@@ -57,14 +62,17 @@
 	update_icon()
 
 /obj/item/weapon/gun/projectile/automatic/ak47/fs
-	name = "FS AR .30 \"Kalashnikov\""
+	name = "FS AR .30 \"Vipr\""						// Vipr like a play on Viper and Vepr
 	desc = "Weapon of the oppressed, oppressors, and extremists of all flavours. \
 			This is a copy of an ancient semi-automatic rifle chambered for .30 Rifle. If it won't fire, percussive maintenance should get it working again. \
 			It is known for its easy maintenance, and low price. This gun is not in active military service anymore, but has become ubiquitous among criminals and insurgents. \
-			This copy, in fact, is a reverse-engineered poor-quality copy of a more perfect copy of an ancient rifle, therefore it can fire only in bursts instead of auto-fire."
+			This copy has been made to look as least miltaristic as possible to be sold to as many civilian populations as possible."
 	icon_state = "AK_wood"
 	item_suffix = "_wood"
 	wielded_item_state = "_doble_wood"
+	w_class = ITEM_SIZE_HUGE
+	recoil_buildup = 1.5	//Heavy, semi-auto and burst only
+	damage_multiplier = 1.1	//Long barrels help you get more out of a round.
```

Comment:
Shouldn't longer, heavier barrels reduce recoil?

Reply:
Unless I am mistaken, isn't a lower recoil better? As in a lower number in this slot?

Reply:
oh I see what you mean now. Possibly, but the longer barrel would also help give the bullet more velocity, which would by newton's third law impart more recoil into the gun, most likely resulting in no net gain or loss. But this can also be up to the balance people if it should be more or less